apicapi/config.py

```diff
@@ -434,7 +435,9 @@ def create_switch_dictionary():
                 switch_dict[switch_id]['pod_id'] = port[0]
                 continue
             hosts = host_list.split(',')
-            hosts = map(lambda a: a.decode('string_escape'), hosts)
+            import codecs
+            hosts = list([codecs.getdecoder('unicode_escape')(a)[0]
```

Comment:
No need to apply list() here as [codecs.getdecoder('unicode_escape')(a)[0] for a in hosts] is already a list. We should also test it out to make sure this does what we expect.

Reply:
Here I agree I was bit over-cautious. I will edit this line.